### PR TITLE
FAQ: luca compatibility note

### LINKED ISSUE #2056: FAQ: Mention that only luca codes generated by businesses (not the user-generated ones) are compatible with CWA
## Where to find the issue
FAQ

## Describe the issue / Suggested change
The FAQ should mention, that QR codes generated by users in their Android/iOS luca-apps ("Meinen QR-Code anzeigen" as well as "privates Treffen erstellen") use different formats that are not compatible with the CWA. Only codes generated by businesses at https://app.luca-app.de/login are compatible.

### PR DESCRIPTION
**Pull request by floscher**
Mentions that these luca QR codes are incompatible:
* any codes generated in the 'luca' app to be scanned with a scanner device owned by the location
* any codes generated in the 'luca' app to be scanned by other 'luca' app users (private meetings)
* any codes generated before May 26th 2021
* any codes generated in the 'luca locations' app, which is intended to be used on devices owned by the location for checking in guests (see the [GitLab issue for that](https://gitlab.com/lucaapp/android/-/issues/67))

TL;DR: Only printed QR codes that were generated by business owners in the web interface after May 25th 2021are compatible.

Fixes #2056 .
Related: #2088 .

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -844,7 +844,9 @@
...
                             "<b>6. Can the Corona-Warn-App read and process the QR codes from luca?</b>",
-                            "Yes. The Corona-Warn-App can scan QR codes issued for event registration using the luca app and perform its own check-in process from the CWA.",
+                            "Yes. The Corona-Warn-App can scan printed out QR codes issued by business owners for event registration with luca and perform its own CWA check-in process.",
+                            "The QR codes shown inside the luca app on smartphones (for private meetings and for check-in through a scanner owned by the location) have different use-cases than and are therefore not supported by the CWA.",
```
Comment (MikeMcC399) [inline src/data/faq.json:848]: "have different use-cases than"
than "what" is missing in the sentence

File: src/data/faq.json [hunk 2]
```diff
@@ -844,7 +844,9 @@
...
                             "<b>6. Can the Corona-Warn-App read and process the QR codes from luca?</b>",
-                            "Yes. The Corona-Warn-App can scan QR codes issued for event registration using the luca app and perform its own check-in process from the CWA.",
+                            "Yes. The Corona-Warn-App can scan printed out QR codes issued by business owners for event registration with luca and perform its own CWA check-in process.",
+                            "The QR codes shown inside the luca app on smartphones (for private meetings and for check-in through a scanner owned by the location) have different use-cases than and are therefore not supported by the CWA.",
+                            "Also, luca decided to use an older incompatible type of QR codes in the 'luca locations' app. So any QR code that is shown by the app on devices of location owners are unfortunately not compatible.",
```
Comment (MikeMcC399) [inline src/data/faq.json:849]: suggest to remove "unfortunately" and just keep to the facts

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
See in-line comments.

File: src/data/faq.json (hunk 2 above)
Comment (floscher) [inline src/data/faq.json:849]:
```suggestion
                            "Also, luca decided to use an older incompatible type of QR codes in the 'luca locations' app. So any QR code that is shown by the app on devices of location owners are not compatible.",
```

File: src/data/faq.json (hunk 1 above)
Comment (floscher) [inline src/data/faq.json:848]: Two possible suggestions:

Either drop the "than" completely (analogous to the German translations):
```suggestion
                            "The QR codes shown inside the luca app on smartphones (for private meetings and for check-in through a scanner owned by the location) have different use-cases than checking in to events and are therefore not supported by the CWA.",
```
Or add to what it's different:
```suggestion
                            "The QR codes shown inside the luca app on smartphones (for private meetings and for check-in through a scanner owned by the location) have different use-cases and are therefore not supported by the CWA.",
```

**Review by floscher — COMMENTED** [review]
Thanks for catching that, I added change suggestions.

File: src/data/faq.json (hunk 1 above)
Comment (MikeMcC399) [reply src/data/faq.json:848]: The second, simpler, version is better i.e. just delete "than".

**Review by MikeMcC399 — APPROVED** [review]
Thanks for making the changes!

**Review by luisoos — APPROVED** [review]

**Comment by MikeMcC399** [comment]
- According to https://www.luca-app.de/luca-digitale-plattform-fuer-buergerinnen/ luca has gone into stand-by.

- So probably this PR should be closed. See also #2751
